src/CoreWCF.Http/tests/SimpleWSHTTPTest.cs

```diff
@@ -233,6 +244,7 @@ internal class WSHttpNoSecurity : StartupWSHttpBase
         {
             public WSHttpNoSecurity() : base(CoreWCF.SecurityMode.None,MessageCredentialType.None)
             {
+               
```

Comment:
Remove extra blank line. With a an empty body I prefer to put them on the same line as the definition. E.g.
```c#
public WSHttpNoSecurity() : base(CoreWCF.SecurityMode.None,MessageCredentialType.None) { }
```